op-node/rollup/derive/doc.go

```diff
@@ -3,14 +3,13 @@
 // turned back into L1 data.
 //
 // The flow is data is as follows
-// receipts, batches -> l2.PayloadAttributes with `payload_attributes.go`
-// l2.PayloadAttributes -> l2.ExecutionPayload with `execution_payload.go`
-// L2 block -> Corresponding L1 block info with `l1_block_info.go`
+// receipts, batches -> eth.PayloadAttributes, by parsing the L1 data and deriving L2 inputs
+// l2.PayloadAttributes -> l2.ExecutionPayload, by running the EVM (using an Execution Engine)
+// L2 block -> Corresponding L1 block info, by parsing the first deposited transaction
```

Comment:
Should we indicate the functions used to performe the transformations?

Reply:
We should long-term, but they're changing with remaining batch-derivation changes, so I'm going for something more generic that doesn't break.